lib/db/file_handler.rb

```diff
@@ -0,0 +1,32 @@
+require 'json'
+
+class FileHandler
+
+  def initialize(file)
+    @file = file
+  end
+
+  def read
+    file.rewind
+    parse(file.read)
+  end
+
+  def write(contacts)
+    file.truncate(0)
+    file.write(contacts.to_json)
+    file.flush
+  end
+
+  def close
+    file.close if file
```

Comment:
I can not find a test for that `if`.
And under which circumstances is it possible that `file` is not initialised?
If there are any, they need to be documented in the tests.